apps/builder/app/builder/features/style-panel/sections/text-shadows/text-shadows.tsx

```diff
@@ -0,0 +1,112 @@
+import { CollapsibleSectionRoot } from "~/builder/shared/collapsible-section";
+import type { SectionProps } from "../shared/section";
+import type {
+  LayersValue,
+  StyleProperty,
+  TupleValue,
+} from "@webstudio-is/css-engine";
+import { useState } from "react";
+import {
+  SectionTitle,
+  SectionTitleButton,
+  SectionTitleLabel,
+  Tooltip,
+  Text,
+} from "@webstudio-is/design-system";
+import { InfoCircleIcon, PlusIcon } from "@webstudio-is/icons";
+import { addLayer } from "../../style-layer-utils";
+import { parseShadow } from "@webstudio-is/css-data";
+import { getDots } from "../../shared/collapsible-section";
+import { PropertyName } from "../../shared/property-name";
+import { getStyleSource } from "../../shared/style-info";
+import { LayersList } from "../../style-layers-list";
+import { ShadowLayer } from "../../shared/shadow-layer";
+
+export const properties = ["textShadow"] satisfies Array<StyleProperty>;
+
+const property: StyleProperty = properties[0];
+const label = "Text Shadows";
+const INITIAL_TEXT_SHADOW = "0px 2px 5px rgba(0, 0, 0, 0.2)";
```

Comment:
what's the difference between this and label, why is this one capital latters? Typically people use capital letters when they need to export and import from different modules so that its always clear its a constant, in this case its going to be always local

Reply:
Initially, i didn't keep them separate. Just followed from the comment here
https://github.com/webstudio-is/webstudio/pull/2955#discussion_r1515908524

Reply:
right, my feeling is to standardize on camel case in all local cases